packages/core/src/components/sections/ProductDetails/ProductDetails.tsx

```diff
@@ -24,9 +22,7 @@ import {
   __experimentalShippingSimulation as ShippingSimulation,
 } from 'src/components/sections/ProductDetails/Overrides'
 
-interface ProductDetailsContextProps {
-  context: ClientProductQueryQuery['product']
-}
+import { usePDP } from 'src/sdk/overrides/PageProvider'
```

Comment:
Right! we will use [this PR](https://github.com/vtex/faststore/pull/1975) for creating the docs.
Do you think it would be good to mention that this section is only used in PDP in the docs?